app/assets/v2/js/grants/detail.js

```diff
@@ -190,3 +190,22 @@ const copyDuplicateDetails = () => {
     });
   });
 };
+
+const notifyOwnerAddressMismatch = () => {
```

Comment:
can we DRY this with the other `notifyOwnerAddressMismatch`?

Reply:
Done 👍  thanks for catching that